### ISSUE TITLE
[Bug] FATAL Rendering entity in world

### ISSUE THREAD

**Issue by zndrmn** [shiroroku/LootBeams#6]
Minecraft Version: 1.16.5
OptiFine Version: G8
Forge Version: 36.2.8
CraftTweaker: 7.1.2.455
Mod Version: lootbeams-1.16.5-release-sept0621.jar

Config:
https://gist.github.com/zndrmn/9dc4332b3258f1d654a034d1fd9cc960

Logs:
Latest.log
https://gist.github.com/zndrmn/bb71b63eaedea510e9ee6d4cf1143aca
crafttweaker.log
https://gist.github.com/zndrmn/292ce7dcd3bdfa0cc918dd25a872517c

Crash Report:
https://gist.github.com/zndrmn/eb7d30eef45441e86cf9ce780010d533

**Comment by shiroroku** [comment]
yeah using non-vanilla rarities seems to cause real weird crashes with lootbeams. newest update renders the beam as white instead of fully crashing the game. ill try to find a fix when i get the time
